From aaa3d45af9c23d490b1d9e448ae5f11159b20893 Mon Sep 17 00:00:00 2001
From: Alessandro Astone <ales.astone@gmail.com>
Date: Wed, 20 Mar 2019 21:10:54 +0100
Subject: [PATCH 09/11] common: recovery: fix reboot

 avc:  denied  { read } for  pid=573 comm="recovery"
 scontext=u:r:recovery:s0 tcontext=u:r:recovery:s0
 tclass=netlink_kobject_uevent_socket permissive=1

Change-Id: Ie28453602be3b5c3c4064057f5f071ee7508d316
---
 common/private/recovery.te | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/common/private/recovery.te b/common/private/recovery.te
index fe4d63c..97d3c1e 100644
--- a/common/private/recovery.te
+++ b/common/private/recovery.te
@@ -28,4 +28,7 @@ set_prop(recovery, system_prop)
 # Switch to backuptool
 allow recovery self:process setexec;
 domain_trans(recovery, otapreopt_chroot_exec, backuptool)
+
+# Fix reboot?
+allow recovery self:netlink_kobject_uevent_socket create_socket_perms_no_ioctl;
 ')
-- 
2.17.1

